### ISSUE TITLE
Elytra Suggestions

### ISSUE THREAD

**Issue by GeorgeZn** [Growlith1223/ArsMagica2#73]
After defeating the ender dragon and finding an end ship, I was really exited to finally use the elytra, and maybe even replace my chest piece of the mage armor, only to find out that it doesn't have the experience imbuement system integrated, not really a big deal but I even thought of some imbuements for it for the level 4 imbuement like repairability with your own mana, automatic take off and stuff like that. (sorry if I misspelled anything, also I don't know if I'm posting this in the right place)

**Comment by Zieg777** [comment]
I have to say, playing with the elytra is a lot of fun. Then I realize I'm not in my earth armor and can actually take damage..

An interesting idea might be to make a glide spell component. The materials required could be an elytra and maybe a rune or something. Ideally it'd last longer than the default 30 seconds.

Might even make it just cost an air essence and some random foci instead of an elytra and make it a more early-game spell. It'd pair nicely with jump boost or a slipstream generator for early game mobility. As it is, levitation seems to be full creative flight anyway.

**Comment by GeorgeZn** [comment]
Yeah, speaking of jump boost I imbued a pair of leggings with the jump boost thingy and well, I take fall damage everytime I jump, if it weren't for the fact that my boots have feather falling I may have died fighting the ender dragon. Isn't it supposed to be like the spell effect?

**Comment by Growlith1223** [comment]
just a heads up, im not doing any new features for now, there's still far too many bugs to be working on something new lol

You can still post suggestions and whatnot, but don't expect them to be added until a lot of the major bugs are fixed
